src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -873,23 +873,6 @@ public GHLicense getLicense(String key) throws IOException {
         return allMyTeams;
     }
 
-    /**
-     * Gets a sigle team by ID.
-     *
-     * @param id
-     *            the id
-     * @return the team
-     * @throws IOException
-     *             the io exception
-     *
-     * @deprecated Use {@link GHOrganization#getTeam(long)}
-     * @see <a href= "https://developer.github.com/v3/teams/#get-team-legacy">deprecation notice</a>
-     */
-    @Deprecated
-    public GHTeam getTeam(int id) throws IOException {
-        return createRequest().withUrlPath("/teams/" + id).fetch(GHTeam.class).wrapUp(this);
-    }
```

Comment:
I decided to remove this method as even if it could work for old GH Entreprise instances, you would end up with a dead end and an object that can't do anything useful.

I can adjust if you prefer I keep it and throw an exception or if you prefer keeping it altogether.

Reply:
We must preserve API compatibility.  Feel free to have it throw if needed.